test/worker_test.rb

```diff
@@ -891,15 +891,11 @@ def self.perform
   end
 
   it "won't fork if ENV['FORK_PER_JOB'] is false" do
-    workerA = Resque::Worker.new(:jobs)
-
-    if workerA.will_fork?
-      begin
-        ENV["FORK_PER_JOB"] = 'false'
-        assert !workerA.will_fork?
-      ensure
-        ENV["FORK_PER_JOB"] = 'true'
-      end
+    begin
+      ENV["FORK_PER_JOB"] = 'false'
+      assert_equal false, Resque::Worker.new(:jobs).fork_per_job?
+    ensure
+      ENV["FORK_PER_JOB"] = 'true'
```

Comment:
nitpick: Shouldn't we reset this to the previous value (rather than resetting it to true)?